samples/msal-angular-v3-samples/angular-standalone-sample/src/main.ts

```diff
@@ -0,0 +1,119 @@
+import { enableProdMode, importProvidersFrom } from '@angular/core';
+import { HTTP_INTERCEPTORS, withInterceptorsFromDi, provideHttpClient } from '@angular/common/http';
+import { MatMenuModule } from '@angular/material/menu';
+import { MatListModule } from '@angular/material/list';
+import { MatToolbarModule } from '@angular/material/toolbar';
+import { MatButtonModule } from '@angular/material/button';
+import { provideNoopAnimations } from '@angular/platform-browser/animations';
+import { BrowserModule, bootstrapApplication } from '@angular/platform-browser';
+import { Route, provideRouter, withDisabledInitialNavigation, withEnabledBlockingInitialNavigation } from '@angular/router';
+import { MsalInterceptor, MSAL_INSTANCE, MsalInterceptorConfiguration, MsalGuardConfiguration, MSAL_GUARD_CONFIG, MSAL_INTERCEPTOR_CONFIG, MsalService, MsalGuard, MsalBroadcastService } from '@azure/msal-angular';
+import { IPublicClientApplication, PublicClientApplication, InteractionType, BrowserCacheLocation, LogLevel, BrowserUtils } from '@azure/msal-browser';
+import { AppComponent } from './app/app.component';
+import { HomeComponent } from './app/home/home.component';
+import { ProfileComponent } from './app/profile/profile.component';
+import { FailedComponent } from './app/failed/failed.component';
+import { environment } from './environments/environment';
+
+const isIE = window.navigator.userAgent.indexOf("MSIE ") > -1 || window.navigator.userAgent.indexOf("Trident/") > -1;
```

Comment:
Any isIE references should be gone from our code in our MSAL.js v3 release.  We're dropping IE support in v3.

Reply:
Makes sense. I've removed all references to it. Thanks!